packages/frontend/package.json

```diff
@@ -55,6 +55,7 @@
         "eslint-config-next": "14.2.15",
         "eslint-config-prettier": "^9.1.0",
         "eslint-plugin-prettier": "^5.2.1",
+        "pino-pretty": "^11.3.0",
```

Comment:
What do we need this for?

Reply:
It suppresses an extremely annoying warning about a missing dependency that Next throws all the time